vendor/github.com/hashicorp/go-sockaddr/ipv4addr.go

```diff
@@ -58,7 +58,8 @@ func NewIPv4Addr(ipv4Str string) (IPv4Addr, error) {
 	// Strip off any bogus hex-encoded netmasks that will be mis-parsed by Go.  In
 	// particular, clients with the Barracuda VPN client will see something like:
 	// `192.168.3.51/00ffffff` as their IP address.
-	trailingHexNetmaskRe := trailingHexNetmaskRE.Copy()
+	// trailingHexNetmaskRe := trailingHexNetmaskRE.Copy()
```

Comment:
Is this a monkey patch? Why was this needed?

Reply:
Yes, for some reason this creates a data race which made tests fail randomly at high concurrency. I still need to understand why that is.